nodestream/project/project.py

```diff
@@ -71,6 +66,7 @@ def describe_yaml_schema(cls):
                 Optional("targets"): {
                     str: {str: object},
                 },
+                Optional("migrations"): os.path.exists,
```

Comment:
Remove this

Reply:
Why?

Reply:
This validates the schema of the project file. I was intending on adding an option in there to customize where the migrations are stored but I decided to keep that out of scope for this PR.